data.sql

```diff
@@ -23,3 +23,25 @@ UPDATE animal SET owners_id = '2' WHERE name = 'Gabumon' OR name = 'Pikachu';
 UPDATE animal SET owners_id = '3' WHERE name = 'Devimon' OR name = 'Plantmon';
 UPDATE animal SET owners_id = '4' WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
 UPDATE animal SET owners_id = '5' WHERE name = 'Angemon' OR name = 'Boarmon';
+
+INSERT INTO vets(name,age,date_of_graduation) VALUES('William Tatcher',45,'2000-04-23'),('Maisy Smith',26,'2019-01-17'), ('Stephanie Mendez',64,'1981-05-04'),('Jack Harness',38,'2008-06-08');
+
+INSERT INTO visits(animals_id,vets_id,date_of_visit) VALUES(1,1,'2020-05-24'),(1,4,'2020-07-22'),(2,3,'2021-02-02'),(3,2,'2020-01-05'),(3,2,'2020-03-08'),(3,2,'2020-05-14'),(4,4,'2021-05-04'),(5,3,'2021-02-24'),(6,2,'2019-12-21'),(6,1,'2020-08-10'),(6,2,'2021-04-07'),(7,4,'2019-09-29'),(8,3,'2020-10-03'),(8,3,'2020-11-04'),(9,2,'2019-01-24'),(9,2,'2019-05-15'),(9,2,'2020-02-27'),(9,2,'2020-08-03'),(10,4,'2020-05-24'),(10,1,'2021-01-11');
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/228926013-4d804999-58d0-4fbf-881a-d9fb0e2136da.png)

- [ ] It would be very great if you add the correct id at the correct place because you have made confusion between id 3 and id 4 where you were supposed to put 4 there your put 3 and where you were supposed to put 3 there you put 4 😨 Jack Harness has id 4, not 3 and Stephanie Mendez has id 4 not 3 🤔